app/integration_test/app_test.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:sharezone/main/run_app.dart';
+import 'package:sharezone/main/sharezone.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  AppDependencies dependencies;
+  _UserCredentials user1;
+
+  setUpAll(() async {
+    dependencies = await initializeFlutterApp(isIntegrationTest: true);
+  });
+
+  setUp(() async {
+    // Credentials are passed via environment variables. See "README.md" how to
+    // pass the them correctly.
+    user1 = _UserCredentials(
+      email: const String.fromEnvironment('USER_1_EMAIL'),
+      password: const String.fromEnvironment('USER_1_PASSWORD'),
+    );
+
+    // We should ensure that the user is logged out before running a test, to
+    // have fresh start.
+    await dependencies.blocDependencies.auth.signOut();
```

Comment:
For the future we might make our own test api `IntegrationTestFixture` for common behaviors like e.g. logging out so that we don't couple our tests too much to the implementation.   

`await testFixture.auth.logOut();` (`auth` would probably also be our own class, not `FirebaseAuth`?) Idk. Let's see when we implement further tests.